.github/workflows/publish.yml

```diff
@@ -24,9 +24,3 @@ jobs:
         run: npm publish
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - name: Build for alias 🏗️
```

Comment:
I would leave it for us as testing branch, but let's discuss on daily

Reply:
I created `publish-alias.yml` workflow, this way we can publish to testing without publishing to main:D wdyt?

7e98f89ef689bafdd6128d9815fc524e2e32ccd1

Reply:
Yeah, I planned to do that, but we still need to figure out versioning.

Reply:
As we discussed, I creates `scripts/bump-version-dev.ts` script that bumps patch version and adds suffix `-dev.${unix timestamp}`. I added it as a step to `publish-dev.yml` and restored `publish.yml` to the way it was so far. This way we have two events when packages are published:

1. When PR is merged to main, `publish-dev.yml` triggers and publishes to `react-native-ci-cli` with dev version. E.g. if on main the version is `0.4.10`, it will release as `react-native-ci-cli@0.4.11-dev.${unix timestamp}`. No new tags/releases are created on GitHub.

2. `release.yml` can be triggered manually, then new GitHub release is created and both `setup-ci` and `react-native-ci-cli` are released with this new version